wow/core/src/main/scala/wow/realm/world/WorldState.scala

```diff
@@ -21,16 +21,24 @@ class WorldState(override implicit val realm: RealmContextData) extends Actor wi
   realm.eventStream.subscribe(self, classOf[PlayerJoined])
   realm.eventStream.subscribe(self, classOf[Tick])
 
-  var currentTick: Tick = Tick(0, Application.uptimeMillis(), Tick(0, 0, null))
-  scheduler.schedule(tickInterval, tickInterval,
+  private var currentTick: Tick = Tick(0, Application.uptimeMillis(), Tick(0, 0, null))
```

Comment:
Same comment about privates and underscores

Reply:
Same as above